Libplanet/Net/IceServer.cs

```diff
@@ -61,7 +61,7 @@ public class IceServer
 
                         return turnClient;
                     }
-                    catch (SocketException)
+                    catch (Exception)
```

Comment:
How about catching both `SocketException` and `ArgumentException` instead of using wildcard here?